tutorials/textUiTestingTutorial.md

```diff
@@ -67,4 +67,8 @@
 1. Update the `javac` and `java` commands in the script to match the name/location of your main class.
 1. Add an `EXPECTED.txt` to the same folder, containing the expected output.
 1. Add an `input.txt` containing the input commands.
-1. Run the `.bat`/`.sh` file to execute the test.
\ No newline at end of file
+1. Run the `.bat`/`.sh` file to execute the test.
+
+---
+**Authors:**
+* Initial Version: Damith C. Rajapakse
```

Comment:
I wrote it based on your work. Anyway, my contributions are too numerous and all over the place, so I'd rather not list my name anywhere specifically.
```
**Authors:**
* Initial Version: based on se-edu/addressbook-level2, adapted by Jeffry Lum
```

Reply:
Got it.